soc/intel/intel_adsp/cavs/Kconfig.series

```diff
@@ -1,8 +1,9 @@
-# Copyright (c) 2017,2022 Intel Corporation
+# Copyright (c) 2017,2022-2024 Intel Corporation
```

Comment:
All `Kconfig.series` files should be renamed to `Kconfig.soc`

Reply:
changed